### DISCUSSION TITLE
Sum of columns

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7748] [Help] [answered]
I am not able to get sum of all the columns where I have used formula type of data in the column

**Comment by melohagan** [comment, marked as answer]
Formula data types apply on a per row basis.

It definitely would be nice to have this though. It has been raised here: https://github.com/Budibase/budibase/issues/3273

For the time being, if you are using an external DB, you could use custom queries to get the totals, and then display those at the bottom of your table. 
Internal DB views can do a Sum calculation. https://docs.budibase.com/docs/views
